website1/dist/index.html

```diff
@@ -402,6 +402,122 @@
       </div>
     </div>
   </section>
+  <section class="stages-wrapper">
+    <div class="container mx-auto">
+      <div class="stages">
+        <div class="stages__title">
+          <div class="title">
+            <p class="title__header">Етапи співпраці</p>
+            <hr class="title__line"/>
+            <p class="title__description">
+              Шлях до безпеки
+            </p>
+          </div>
+        </div>
+        <div class="stages-box">
+          <div class="stages-box__item">
+            <div class="item__number">
+              <p class="item__number--number">
```

Comment:
`--` - it's what?

Reply:
common box inside which are boxes of elements